test/sql/updates/post.catalog.sql

```diff
@@ -30,10 +30,43 @@ ORDER BY schema, name, initpriv;
 
 \di _timescaledb_catalog.*
 \ds+ _timescaledb_catalog.*
-\df _timescaledb_internal.*
-\df+ _timescaledb_internal.*
-\df public.*;
-\df+ public.*;
+
+-- Functions in schemas:
+--   * _timescaledb_internal
+--   * _timescaledb_functions
+--   * public
+SELECT n.nspname as "Schema",
+  p.proname as "Name",
+  pg_catalog.pg_get_function_result(p.oid) as "Result data type",
+  pg_catalog.pg_get_function_arguments(p.oid) as "Argument data types",
+ CASE p.prokind
+  WHEN 'a' THEN 'agg'
+  WHEN 'w' THEN 'window'
+  WHEN 'p' THEN 'proc'
+  ELSE 'func'
+ END as "Type",
+ CASE
+  WHEN p.provolatile = 'i' THEN 'immutable'
+  WHEN p.provolatile = 's' THEN 'stable'
+  WHEN p.provolatile = 'v' THEN 'volatile'
+ END as "Volatility",
+ CASE
+  WHEN p.proparallel = 'r' THEN 'restricted'
+  WHEN p.proparallel = 's' THEN 'safe'
+  WHEN p.proparallel = 'u' THEN 'unsafe'
+ END as "Parallel",
+ pg_catalog.pg_get_userbyid(p.proowner) as "Owner",
+ CASE WHEN prosecdef THEN 'definer' ELSE 'invoker' END AS "Security",
+ CASE WHEN pg_catalog.array_length(p.proacl, 1) = 0 THEN '(none)' ELSE pg_catalog.array_to_string(p.proacl, E'\n') END AS "Access privileges",
+ l.lanname as "Language",
+ p.prosrc as "Source code",
+ CASE WHEN l.lanname IN ('internal', 'c') THEN p.prosrc END as "Internal name",
+ pg_catalog.obj_description(p.oid, 'pg_proc') as "Description"
+FROM pg_catalog.pg_proc p
+     LEFT JOIN pg_catalog.pg_namespace n ON n.oid = p.pronamespace
+     LEFT JOIN pg_catalog.pg_language l ON l.oid = p.prolang
+WHERE n.nspname OPERATOR(pg_catalog.~) '^(_timescaledb_internal|_timescaledb_functions|public)$' COLLATE pg_catalog.default
+ORDER BY 1, 2, 4;
```

Comment:
Not sure a 30 line sql command is an improvement over a 4 line psql meta commands

Reply:
The \df+ output changed so I decided to hardcode this output on the test

Reply:
i think \df is much easier to maintain, also this seems unrelated to the ci fix so we should probably separate this and get the ci fix in quickly

Reply:
It is not unrelated because before we ran the psql from the newest version against the old cluster to produce exact the same output and don't have this change. Not `PostgresNode::port` is read-only so we can't change the port on-the-fly to run psql from newest cluster against oldest. O also think that `\df` is way easier to maintain but this `post.catalog.sql` have `\df+` since forever. Do u think it is not necessary?

Reply:
hmm I think \df+ is useful to verify the result of update scripts, if we run the same psql version against both versions there should be no diff, right?

Reply:
> hmm I think \df+ is useful to verify the result of update scripts, if we run the same psql version against both versions there should be no diff, right?

This is exactly what I tried to explain because now the `PostgresNode::port` is read-only so I can't change it in order to run new psql version against old cluster. Check the .py changes.